### ISSUE TITLE
Problem with touch screen devices

### ISSUE THREAD

**Issue by anandchapter247** [payalord/xZoom#43]
I've integrated this plugin in my web app and it's working fine in desktop, but issue occurred while doing the same with touch devices. How can I integrate this plugin with touchscreen device, Please help me regarding same?

**Comment by payalord** [comment]
Because there is a lot libraries that support touch gestures, when I created xZoom I decided not to implement my own into xZoom, instead create as much as possible API functions with which everyone will be able to integrate it with touch gestures or anything else, their preferred way. If you take a look here from mobile device it works with touch screen (tap is opening, then when it is opened double tap opening MagnificPopup or Fancybox in their examples, plus movement is working too):
https://payalord.github.io/xZoom/

On that example page, xZoom integrated with HammerJS version 1. But it is already HammerJS version 2 available. But I don't have enough time to create normal examples in xzoom-sandbox for this.
